### PR TITLE
[ENG-4551] Cesium/third batch of bug fixes

### PR DESCRIPTION
**Pull request by brianjgeiger**
-   Ticket: [ENG-4551]
-   Feature flag: n/a

## Purpose

Fix some more things found by QA. Best reviewed commit-by-commit.

## Summary of Changes

See commit list

## Screenshots

Screenshot 2023-06-22 at 9 41 51 AM
Screenshot 2023-06-23 at 1 17 07 PM
Screenshot 2023-06-22 at 2 00 45 PM
Screenshot 2023-06-23 at 1 19 32 PM
Screenshot 2023-06-23 at 1 20 19 PM
Screenshot 2023-06-23 at 1 21 14 PM

[ENG-4551]: https://openscience.atlassian.net/browse/ENG-4551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/registries/addon/components/registries-recent-list/styles.scss [hunk 1]
```diff
@@ -5,6 +5,8 @@
     border-bottom: 1px solid #e0e6ea;
     padding-bottom: 15px;
     padding-top: 15px;
+    padding-left: 50px;
+    padding-right: 50px;
```
Comment (futa-ikeda) [inline lib/registries/addon/components/registries-recent-list/styles.scss:9]: Applying this to staging2, it looks like these paddings are more appropriate on the `.recent-registries-container` class

File: lib/osf-components/addon/components/osf-navbar/auth-dropdown/template.hbs [hunk 2]
```diff
@@ -11,7 +11,7 @@
                 <img src='{{this.user.profileImage}}&s=25' alt='{{t 'auth_dropdown.user_gravatar'}}'>
             </div>
             <div class='nav-profile-name'>{{this.user.fullName}}</div>
-            <span class='caret'></span>
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/osf-navbar/auth-dropdown/template.hbs:14]: I'm glad this is now the "old way" of doing this 👀

**Review by futa-ikeda — CHANGES_REQUESTED** [review]
Had a startle when I checked the settings page at first! Just one rule that I think belongs on a different element

File: lib/osf-components/addon/components/osf-navbar/auth-dropdown/template.hbs (hunk 2 above)
Comment (brianjgeiger) [reply lib/osf-components/addon/components/osf-navbar/auth-dropdown/template.hbs:14]: Agreed

**Review by futa-ikeda — APPROVED** [review]

**Pull request merged**